styles/style.css

```diff
@@ -92,12 +104,24 @@ p {
   border-left-color: var(--orange-dark);
 }
 
-
 /* Buttons */
+button {
+  background-color: #c05326;
+  color: white;
+  border-radius: 5px;
+}
 
-  /* INSERT BUTTON STYLES HERE */
+/* INSERT BUTTON STYLES HERE */
 
+button:hover {
+  background-color: gray;
+}
 
+.btn {
+  background-color: white;
+  color: orange;
```

Comment:
For the sake of the page looking more synchronous, you always want to use similar colors. This orange doesn't match the one we used for the other button! Play around with the colors and the font-weight of this!!